src/ui/include/EditorNS/editor.h

```diff
@@ -290,9 +257,11 @@ namespace EditorNS
         void setSelection(int fromLine, int fromCol, int toLine, int toCol);
 
         int lineCount();
+        int textLength();
```

Comment:
While we're at it, should we use `getLineCount()` or just `lineCount()` for getters? Because right now we're rather inconsistent with that. @danieleds?

IMO, `getLineCount()` is better since it plays very nicely with autocomplete.